src/lib.zig

```diff
@@ -0,0 +1,11 @@
+const std = @import("std");
+const mem = std.mem;
+const Allocator = std.mem.Allocator;
+
+export fn roc_check(path: *const c_char) i32 {
```

Comment:
If we make a full featured lib roc, we should probably make `main.zig` strictly build roc via the same interfaces as `libroc`. That ensures they stay in sync.

That said, I'm not sold we want a full featured lib roc. I think we likely want a super small shim lib roc that only has the ability to interact with the intepreter.